scripts/azure-windows.yml

```diff
@@ -12,6 +12,12 @@ steps:
     fi
 
 - powershell: |
+
+    $env:Path += ";c:\msys64\usr\bin"
```

Comment:
Is this still necessary? Need to be careful because cmake may choose a different compiler if it finds one in PATH.

Reply:
Removed.